### PR TITLE
fix(eslint-plugin): [prefer-optional-chain] handling first member expression

### PR DESCRIPTION
**Pull request by yeonjuan**
There is a false negative when member expression comes first.

```ts
foo.bar && foo.bar.baz;  // no error
```

### CODE REVIEW

**Comment by yeonjuan** [comment]
Sorry. it should not be merged. cause there are normal cases which should return `null`

**Comment by yeonjuan** [comment]
It seems to affect the current code.
I'm not sure the ignoring first member expression is bug or not. If it's a bug I'll change them.

File: packages/eslint-plugin/src/rules/prefer-optional-chain.ts [hunk 1]
```diff
@@ -55,6 +55,7 @@ export default util.createRule({
     return {
       [[
         'LogicalExpression[operator="&&"] > Identifier',
+        'LogicalExpression[operator="&&"] > MemberExpression',
         'LogicalExpression[operator="&&"] > BinaryExpression[operator="!=="]',
         'LogicalExpression[operator="&&"] > BinaryExpression[operator="!="]',
       ].join(',')](
```
Comment (bradzacher) [inline packages/eslint-plugin/src/rules/prefer-optional-chain.ts:61]: You need to update the node type with this selector change as well

Comment (yeonjuan) [reply packages/eslint-plugin/src/rules/prefer-optional-chain.ts:61]: @bradzacher Oops :) Thanks!

**Pull request merged**